deploy/docker/hail_search/bin/sync_datasets

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+suffixes=(
+    GRCh37_SNV_INDEL_VERSION
+    GRCh38_SNV_INDEL_VERSION
+    GRCh38_MITO_VERSION
+    GRCh38_SV_VERSION
+    GRCh38_GCNV_VERSION
+)
+
+function to_gcs_path () {
+    # e.g. GRCh38_SNV_INDEL_VERSION -> GRCh38/SNV_INDEL
+    echo $1 | sed "s|GRCh38_|GRCh38/|g" | sed "s|GRCh37_|GRCh37/|g" | sed "s|_VERSION||g"
+}
+
+for suffix in "${suffixes[@]}"; do
+    path=$(to_gcs_path $suffix)
+    gsutil -m rsync -r $SOURCE_DATASETS_DIR/$path/runs/${!suffix} $DATASETS_DIR/$path
```

Comment:
for the prototype, I didn't run the copying in the seqr from the hail search image at all, I ran it as an initContainer defined in the yaml file using a gcloud cli docker image: https://github.com/broadinstitute/seqr/compare/dev...hail-backend#diff-1170a92e32cc8b8d3707780992a5ce0046c05f6d2c7e3560c71694ee6c1e4c71R62

Reply:
I feel like templating that out in helm would not be too bad actually

Reply:
Yep, my initial pass was using that image... then I moved the code here because the bash got unwieldy in helm.  I'll give it another shot though, since I think installing `gcloud` into this image isn't great.

Reply:
I [end up with this](https://github.com/broadinstitute/seqr-helm/pull/19/files#diff-5f718a9c818a4d6921db78158471daa121a855a05148515dc9938ebd28711955R35-R48) which I think is the most reasonable thing at this point.

Reply:
we could also store the variables in some sort of dictionary, so the values would look like
```
versions:
  GRCh37/SNV_INDEL: manual__2023-..
  GRCh38/SNV_INDEL: manual__2023-..
  GRCh38/MITO: manual__2023-..
  GRCh38/GCNV: manual__2023-..
  GRCh38/SV: manual__2023-..
```

and then the helm template would look like
```
{{- range $path, $version := .Values.environment.versions }}
  gsutil -m rsync -r {{ .Values.environment.SOURCE_DATASETS_DIR }}/{{ $path }}/runs/{{ version }} {{ .Values.environment.DATASETS_DIR }}/{{ $path }};
{{- end }}
```

Reply:
yep! something like that works.